src/Microsoft.AspNet.Mvc.Core/ResponseCache.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Mvc
+{
+    /// <summary>
+    /// An action filter which sets the appropriate headers related to Response caching.
+    /// </summary>
+    public class ResponseCache : ActionFilterAttribute
```

Comment:
@rynowak should this have AllowMultiple = false? `ActionFilterAttribute` allows multiple